### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by alexAubin**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
 		"email": "maniackc_dev@crudelis.fr"
 	}],
     "requirements": {
-        "yunohost": ">= 4.1.2"
+        "yunohost": ">= 4.3.0"
```
Comment (alexAubin) [inline manifest.json:27]: Note that this change is probably what's causing the CI to fail, as if we require yunohost >= 4.2, this implicitly triggers some `chmod/chown` during `ynh_setup_source`

We probably to tweak permissions (eg allow www-data probably?) to fix it

**Comment by alexAubin** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by kay0u** [comment]
Wut @yalh76 ?

**Comment by yalh76** [comment]
sorry I tried to restart it from scratch, but didn't end ... maybe during this week

**Pull request closed without merging**